.editorconfig

```diff
@@ -150,6 +150,10 @@ dotnet_diagnostic.IDE0060.severity = none
 
 dotnet_diagnostic.IDE0060.severity = none
 
+[test/Mono.Linker.Tests/TestCases/Dependencies/WarningSuppressionExpectations*.cs]
```

Comment:
Is there a reason to exclude just these files, or is it left over?

Reply:
These files are used as 'expectations' of the generated suppressions files by the trimmer. The trimmer produces suppressions files but doesn't include the header on them, therefore adding the header to this expectation files will make a test to fail.